# Pensamiento geométrico para visualizar y argumentar

#### Ejercicio 5

$A$, $B$, $C$, $D$, y $E$ son puntos consecutivos. Las siguientes longitudes se conocen en términos de $x$ y $y$:
 \begin{align*}
 \overline{AB}=&x-y\\
 \overline{AD}=&x+2y-1\\
 \overline{BD}=&\overline{AB}+1\\
 \overline{DE}=&4y-4\\
 \overline{AE}=&2x+y
 \end{align*}
 Encuentra el valor numérico de la longitud de $\overline{AB}$.

#### Solución

Primera ec.:

$$\overline{AB}+\overline{BD}=\overline{AD}$$

$$ (x-y)+(\overline{AB}+1)=(x+2y-1) $$

$$ (x-y)+[(x-y)+1]=(x+2y-1) $$

Segunda ec.:

$$\overline{AD}+\overline{DE}=\overline{AE}$$


$$ (x+2y-1)+(4y-4)=(2x+y) $$

Revisen con algún métdodo de solución de ecuaciones que $x=10$, $y=3$.

# Clase 2

## Círculos I

Una de las preguntas más importantes de las matemáticas:

**¿Qué es $\pi$?**

<img src="circ1.png" align="middle" width="300">

* El punto negro es el *centro* del círculo 
* La línea negra es la *circunferencia* (el perímetro de un círculo).
* La línea roja es el *diámetro* (línea que va de un punto de la circunferencia a otro, pasando por el centro).
* La línea verde es el *radio* (línea que va del dentro hasta un punto de la circunferencia).
* La línea azul es la *cuerda* (línea que va de un punto de la circunferencia a otro sin pasar por el centro).

<img src="circ2.png" align="middle" width="300">

* La línea roja representa un *arco*.
* La sección de color verde se conoce como *sector circular*.
* La sección de color azul se conoce como *segmento circular*.

**¿Qué es $\pi$?**

$\pi$ es la proporción entre circunferencia y el diámetro, o sea, las veces que cabe el diámetro en la circunferencia de un círculo:

$$ \pi=\dfrac{\text{circunferencia}}{\text{diámetro}} $$

In [2]:
π

π = 3.1415926535897...

A diferencia de valores como $\dfrac{1}{3}$ y $\dfrac{1}{7}$:

In [3]:
1/3

0.3333333333333333

In [16]:
1/7

0.14285714285714285

las decimales de $\pi$ nunca se repiten.

### Perímetro

* $D$ : diámetro
* $R$ : radio
* $C$ : circunferencia

$$ C=\pi D $$

El diámetro es el doble del radio:

$$ D=2R $$

Entonces:

$$ C=2\pi R $$

La mitad de una circunferencia es

$$ \dfrac{\pi D}{2}=\dfrac{\pi (2R)}{2}=\pi R $$

### Perímetro arco

<img src="circ2.png" align="middle" width="300">

**¿Cómo hacemos para obtener la longitud de cualquier arco?**

$$ s=\theta R $$

* $s$ : longitud de arco
* $\theta$ : ángulo (medido en radianes)

Para esto diremos que $1$ vuelta es $360^\circ$ y es lo mismo que $2\pi\ \rm{rad}$.

Notemos que:

$$ 2\pi\ \rm{rad}=360^\circ $$

$$ \pi\ \rm{rad}=180^\circ $$

$$ \dfrac{\pi}{2}\ \rm{rad}=90^\circ $$

$$ \dfrac{\pi}{3}\ \rm{rad}=60^\circ $$

$$ \dfrac{\pi}{4}\ \rm{rad}=45^\circ $$

$$ \dfrac{\pi}{6}\ \rm{rad}=30^\circ $$

### Conversión de unidades

* ***Ejemplo:*** $72\,{}^\circ$ a $\rm{rad}$

$$ 72\,{}^\circ=72\,{}^\circ\left(\dfrac{\pi\ \rm{rad}}{180^\circ}\right)=\dfrac{72}{180}\pi\ \rm{rad}=\dfrac{36}{90}\pi\ \rm{rad}=\dfrac{18}{45}\pi\ \rm{rad}=\dfrac{2}{5}\pi\ \rm{rad} $$

In [5]:
2*π/5

1.2566370614359172

In [6]:
2*(3.1416)/5

1.25664

$$ 72\,{}^\circ=1.256637...\ \rm{rad} $$

* ***Ejemplo:*** $144\,{}^\circ$ a $\rm{rad}$

$144^\circ$ es el doble de $77^\circ$.

Entonces, la conversión será el doble de $\dfrac{2}{5}\pi\ \rm{rad}$

$$ 144\,{}^\circ=\dfrac{4}{5}\pi\ \rm{rad} $$

* ***Ejemplo:*** $\dfrac{37}{7}\pi\ {\rm{rad}}$ a ${}^\circ$

$$ \dfrac{37}{7}\pi\ {\rm{rad}}=\dfrac{37}{7}\pi\ {\rm{rad}}\left(\dfrac{180^\circ}{\pi\ \rm{rad}}\right)=\dfrac{37\cdot180}{7}^\circ=951.42^\circ $$

In [18]:
37*180/7

951.4285714285714

Si lo hacemos primero multiplicando $\dfrac{37\pi}{7}$

In [11]:
a = 37*π/7

16.60556116897462

$$ \dfrac{37}{7}\pi\ {\rm{rad}}=16.6055...\ {\rm{rad}} $$

obtenemos lo mismo:

In [12]:
a*180/π

951.4285714285714

### Perímetro arco

* ***Ejemplo:*** Calcular el perímetro de un sector circular de radio $2\ \rm{cm}$ y ángulo de $30\,{}^\circ$.

<img src="seg_circular1.png" align="middle" width="200">

**Calcular el perímetro es preguntarnos cuánto mide un estambre que cubra el entorno del segmento (la línea de color azul rey).**

Primero tenemos que convertir $30^\circ$ a $\rm{rad}$:

$$ 30^\circ=\dfrac{30^\circ\cdot 2\pi\ \rm{rad}}{360^\circ}=\dfrac{30\cdot 2}{360}\pi\ \rm{rad}=\dfrac{6}{36}\pi\ \rm{rad}=\dfrac{\pi}{6}\ \rm{rad}$$

La longitud del arco es:

$$ s=\theta R=\left(\dfrac{\pi}{6}\ \rm{rad}\right)(2)\ \rm{cm} $$

$$ =\dfrac{\pi}{3}\ \rm{cm}=1.0471...\ \rm{cm}$$

In [19]:
π/3

1.0471975511965976

Veamos que el arco tienen $2$ radios a los lados por lo que el perímetro simplemente es

In [20]:
π/3+2+2

5.047197551196597

$$ P=5.0471...\ \rm{cm}$$

$P$ : perímetro

### Área

El área de un círculo es

$$ A=\pi R^2 $$

### Sectores circulares

El perímetro del segmente circular se mide como
$$ P=s+2R=R\theta+2R=R(\theta+2) $$

El área de un sector circular es

$$ A= \dfrac{\theta R^2}{2} $$

**Nota importante:** Aquí el ángulo se mide en radianes.

* ***Ejemplo:*** Calcular el área de un sector circular de radio $2\ \rm{cm}$ y ángulo de $30\,{}^\circ$.

<img src="seg_circular1.png" align="middle" width="200">

$$ \theta=30^\circ=\dfrac{\pi}{6}\ \rm{rad}$$

El área de un sector circular es

$$ A= \dfrac{\theta R^2}{2}= \dfrac{\dfrac{\pi}{6} (2\ \rm{cm})^2}{2}=\dfrac{\dfrac{\pi}{6} 4\ \rm{cm}^2}{2}\ \rm{cm}^2=\dfrac{\pi}{6}\cdot2\ \rm{cm}^2=\dfrac{2\pi}{6}\ \rm{cm}^2=\dfrac{\pi}{3}\ \rm{cm}^2 $$

El área de un segmento circular se mide como
$$ A=\dfrac{R^2}{2}\theta $$

* ***Ejemplo:*** Calcular el área y perímetro de un sector circular de radio 5 y ángulo de $\dfrac{3}{4}\pi\ \rm{rad}$.

## Puntos, rectas y segmentos en dos dimensiones

Cuando se tiene dos rectas que nunca se cruzan decimos que estas son *paralelas*.

<img src="paralelas.png" align="middle" width="500">

Si dos rectas **NO** son paralelas, en algún punto se tienen que intersectar.

### Teorema de Tales

<img src="tales.png" align="middle" width="300">

*Si tenemos tres rectas paralelas y estas cortan a dos rectas que no son paralelas se producen segmentos proporcionales:*

$$ \dfrac{\overline{AB}}{\overline{AC}}=\dfrac{\overline{DE}}{\overline{DF}} $$

También se cumple que

$$ \dfrac{\overline{AB}}{\overline{BC}}=\dfrac{\overline{DE}}{\overline{EF}} $$

### Semejanza

Dos cosas semejantes si son de la misma forma aunque tengan distinto tamaño.

### Triángulos semejantes

Dos triángulos son semejantes si tienen la misma forma pero distinto tamaño. Dicho de otra forma, dos triángulos son semejantes si los triángulos tienen los mismo ángulos y los lados son proporcionales. La manera en que nos aseguramos que sean semejantes es:
* si tienen todos sus ángulos iguales entre sí
* si tienen dos ángulos iguales entre sí
* si tienen un ángulo igual entre sí y sus dos de sus lados son proporcionales con los lados correspondientes del otro triángulo
* si todos sus lados son proporcionales entre sí

***Ejemplo:*** Pensemos que los triángulos semejantes son: 

<img src="semejanza1.png" align="middle" width="500">

Para los lados correspondientes entre sí escribimos:

$$ \dfrac{x+1}{4}=\dfrac{8}{y-1}=\dfrac{12}{6}  $$

**Primera ec.:**

$$ \dfrac{x+1}{4}=\dfrac{12}{6}  $$

$$ \dfrac{x+1}{4}=2  $$

$$ x+1=2\cdot4  $$

$$ x+1=8  $$

$$ x=7  $$

**Segunda ec.:**

$$ \dfrac{8}{y-1}=\dfrac{12}{6}  $$

$$ \dfrac{8}{y-1}=2  $$

$$ 8=2(y-1)  $$

$$ \dfrac{8}{2}=y-1  $$

$$ 4=y-1  $$

$$ y=4+1  $$

$$ y=5  $$

<img src="semejanza1.png" align="middle" width="500">

$$ \dfrac{x+1}{4}=\dfrac{8}{4}=2  $$

$$ \dfrac{8}{y-1}=\dfrac{8}{4}=2  $$

$$ \dfrac{12}{6}=2  $$

Entonces **sí** tienen la misma proporción.

***Ejemplo:*** El gato de Leticia subió a la parte superior de un poste. Leticia puede ver a su gato reflejado en un charco, que se encuentra a $1.6\ \rm{m}$ de ella. Si la altura de sus ojos es de $1.44\ \rm{m}$ y el poste se encuentra a $5.6\ \rm{m}$ de ella, ¿qué tan alto es el poste?